appveyor.yml

```diff
@@ -0,0 +1,21 @@
+# Test against this version of Node.js
```

Comment:
Since jest just tests the javascript module we don't need a windows environment just _yet_ (everything is mocked).

Won't hurt to have this file though. Awesome!!